libs/blocks/chart/chart.js

```diff
@@ -125,6 +126,20 @@ const barSeriesOptions = (chartType, seriesData, colors, size, unit) => {
   }));
 };
 
+const lineSeriesOptions = (seriesData) => (
```

Comment:
It seems like `seriesData` should really be called something related to dataset or source, like `firstDataset` or `firstSourceData`, as it's only indirectly used for the series options.

Reply:
Renamed to firstDataset